redhat-appstudio/service-provider-integration-oauth
# Accomodate for the changes in the operator after removing the sharedSecret

### What does this PR do?
Accomodate for the changes in the operator after removing the sharedSecret
from configuration and the resulting simplification of the oauthstate encoding.

### What issues does this PR fix or reference?
https://issues.redhat.com/browse/SVPI-206

### How to test this PR?
See the corresponding operator PR: https://github.com/redhat-appstudio/service-provider-integration-operator/pull/267
